web/gds-admin-ui/src/components/CountryOptions.js

```diff
@@ -0,0 +1,15 @@
+
+import React from 'react'
+import { isoCountries } from 'utils/country'
+
+function CountryOptions() {
+    return (
+        <>
+            {
+                Object.entries(isoCountries).map(([k, v]) => <option key={k} value={k}>{v}</option>)
```

Comment:
@elysee15 could we also add an empty option so that users can specify "no country" in the form. E.g. in the national identification details, we have to have either country or registration authority.

Reply:
I'd like this to be released today; so I'm going to go ahead and merge this and create a follow on story for stuff like this.